website/header_menu_items.py

```diff
@@ -37,6 +37,10 @@ class HeaderLink:
             HeaderLink(label=_("Our Sponsors"), href="/sponsors"),
         ]
     ),
+    HeaderLink(
+        label=_("Grant"),
+        href="/grant",
```

Comment:
```suggestion
        href="/opportunity_grants",
```
I think this is clearer.